source/apache-way/apache-project-maturity-model.md

```diff
@@ -191,76 +186,129 @@ access or decision power, and applies these principles consistently.
 
 <dt id="CO60">CO60</dt>
 <dd>
-The community operates based on consensus of its members (see CS10) who 
-have decision power. Dictators, benevolent or not, are not welcome in 
-Apache projects. 
+The community operates based on consensus of its members (see CS10) who
+have decision power. Dictators, benevolent or not, are not welcome in
+Apache projects.
 </dd>
 
 <dt id="CO70">CO70</dt>
 <dd>
 The project strives to answer user questions in a timely manner.
 </dd>
+</dl>
+
+## Trademark and Branding
+
+Notable items from the [Trademark Policy][trademark-policy],
+[FAQ][trademark-faq], [Guide][trademark-guide], and
+[PMC Branding Responsibilities][branding-responsibilities].
+The project's PMC should comply with ASF's branding and trademark policy in
+all places it controls.
+
+[trademark-policy]: https://www.apache.org/foundation/marks/
+[trademark-faq]: https://www.apache.org/foundation/marks/faq/
+[trademark-guide]: https://www.apache.org/foundation/marks/guide
+[branding-responsibilities]: https://www.apache.org/foundation/marks/responsibility
+
+<dl>
+<dt id="TB10">TB10</dt>
+<dd>
+The project uses "Apache Foo™" as the project and software product
+name consistently, with appropriate [trademark attributions][trademark-attributions].
+</dd>
 
+[trademark-attributions]: https://apache.org/foundation/marks/pmcs#attributions
+
+<dt id="TB20">TB20</dt>
+<dd>
+The project's primary homepage is at `projectname.apache.org`.
+When exceptions exist to use alternate domains for any purpose, any
+`non-apache.org` domain names are owned by the ASF.
+</dd>
+
+<dt id="TB30">TB30</dt>
+<dd>
+The ASF has trademark rights, including any registrations, to the
+project name, logo, and any other major branding elements.
+
+NB. For example, any transfers of existing trademark registrations
+during Incubation must completed before graduation.
+</dd>
+
+<dt id="TB40">TB40</dt>
+<dd>
+The project monitors for any major misuses of their project's brand
+by others, and [reports][reports-trademark-issues] any potential
+misuses to Brand Management.
+
+NB. TB40 is a balancing act: we hope that PMCs are spending some
+time responding to any obvious or blatant misuses of their brand.
+But PMCs sometimes can't handle all the details of actually stopping a
+potential infringement; what we really need is for PMCs to actively
+report them, and then promptly respond if Brand Managment (or other
+officers) ask for information when addressing a misuse.
+
+[reports-trademark-issues]: https://apache.org/foundation/marks/reporting.html#pmcs
```

Comment:
The replace the original proposed:

```
The project proactively monitors 3rd party use of its trademarks,
addressing any instances of misuse.

NB. For example, regularly/irregularly check how downstream
refer to the project, check how 3rd party sites refer to the project.
```

I suppose it's more actionable and clearly clarify our expectations.